shell/packages/sandstorm-db/scheduled-jobs-db.js

```diff
@@ -0,0 +1,145 @@
+// Sandstorm - Personal Cloud Sandbox
+// Copyright (c) 2017 Sandstorm Development Group, Inc. and contributors
+// All rights reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//   http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+const Crypto = Npm.require("crypto");
+
+const PERIOD_MILLIS = {
+  yearly:  1000 * 60 * 60 * 24 * 365,
+  monthly: 1000 * 60 * 60 * 24 * 30,
```

Comment:
If we're going to go to the trouble of providing symbolic periods instead of just doing a numeric value, we should be smart enough to know that a month isn't always 30 days, leap-years are a thing, etc.